src/cinder/Triangulate.cpp

```diff
@@ -66,15 +73,16 @@ Triangulator::Triangulator()
 void Triangulator::allocate()
 {
 	mAllocated = 0;
-	
-	TESSalloc ma;
-	memset( &ma, 0, sizeof(ma) );
-	ma.memalloc = stdAlloc;
-	ma.memfree = stdFree;
-	ma.userData = (void*)&mAllocated;
-	ma.extraVertices = 2560; // realloc not provided, allow 256 extra vertices.
-
-	mTess = shared_ptr<TESStesselator>( tessNewTess( &ma ), tessDeleteTess );
+
+	mAlloc = shared_ptr<TESSalloc>( new TESSalloc() );
```

Comment:
sorry that was a typo, i meant to write unique_ptr. i agree that makes more sense so i'll try adding a destructor and see if that helps.